cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -121,12 +123,34 @@ public Integer doAction() throws Exception {
     return 0;
   }
 
+  /**
+   * check if Old templates exists to handle the exception and let the user decides, if the templates should be upgraded
+   *
+   *
+   *
+   */
+  private void checkOldTemplatesException() {
+
+    try {
+      CobiGenUtils.initializeCobiGen(this.templatesProject);
+    } catch (DeprecatedMonolithicTemplatesException e) {
+      e.printStackTrace();
+      LOG.info("Would you like to upgrade your templates to the newest version? \n"
+          + "type yes/y to continue or no/n to cancel (or hit return for yes).", System.getProperty("user.dir"));
```

Comment:
We should remove the printStackTrace and add the exception to the info or maybe another warning message.